dotcom-rendering/src/web/components/CPScottHeader.tsx

```diff
@@ -0,0 +1,64 @@
+import { css } from '@emotion/react';
+import { headline, neutral } from '@guardian/source-foundations';
+import { Link } from '@guardian/source-react-components';
+import { Hide } from './Hide';
+import { QuoteIcon } from './QuoteIcon';
+
+const scottAvatarStyles = css`
+	width: 72px;
+	height: 72px;
+	border-radius: 100%;
+	overflow: hidden;
+	background-color: #fcebde; /* stylelint-disable-line color-no-hex */
```

Comment:
Is it worth catching up with @HarryFischer to see if we should use something from our palette here?

Reply:
Is this the background behind his cutout? We can match the pale opinion colour so it matches the card background